.github/workflows/tests.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       - name: Set up miniconda
         uses: conda-incubator/setup-miniconda@v3
         with:
-          miniforge-variant: Mambaforge
```

Comment:
This is necessary now, since support has dropped fully:

https://github.com/fulcrumgenomics/prymer/issues/27

Also implemented in:

- https://github.com/fulcrumgenomics/prymer/pull/32